decoder/pokemon.go

```diff
@@ -368,20 +378,22 @@ func (pokemon *Pokemon) updateFromWild(ctx context.Context, db db.DbDetails, wil
 	pokemon.IsEvent = 0
 	encounterId := strconv.FormatUint(wildPokemon.EncounterId, 10)
 	oldWeather, oldPokemonId := pokemon.Weather, pokemon.PokemonId
+	pokemonId := int16(wildPokemon.Pokemon.PokemonId)
+	pokemon.setPokemonDisplay(pokemonId, wildPokemon.Pokemon.PokemonDisplay)
 
 	if pokemon.Id == "" || pokemon.SeenType.ValueOrZero() == SeenType_Cell || pokemon.SeenType.ValueOrZero() == SeenType_NearbyStop {
 		updateStats(ctx, db, encounterId, stats_seenWild)
 	}
+	if pokemon.IsDitto {
+		// TODO: add some more logic regarding event change, actually a Ditto will remain as Ditto
```

Comment:
What do you mean? Source?

Reply:
When Spotlight hour starts that ditto would probably change to the spotlight hour mon, false? Would it stay a ditto for sure?

Reply:
I am guessing that it should completely reroll. Haven't tested this though.

Reply:
Guess what. Using #65, I caught a Snubbull disguised as a Ditto rerolled into a Houndour (impossible to be Ditto).

```
INFO 2023-03-23 03:52:29 Pokemon 16593934753357378148 changed from (209,1253,0,1) to (228,229,0,1)
INFO 2023-03-23 03:52:29 [POKEMON] 16593934753357378148: B0>00/PP Ditto found, disguised as 228. (4,-4,0/0/0,148)>(0,1,4/9/0)
```

Reply:
a/b spawn I would say :/